client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -232,72 +221,33 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 					fromSite={ fromSite }
 					urlData={ fromSiteData ?? undefined }
 					stepNavigator={ stepNavigator }
-					renderHeading={ ! useContainerV2 }
```

Comment:
same here as in the previous comment

Reply:
I decided against removing this one because it's a prop that is being passed down by a HoC, and there are a bunch of references so I'm hesitant to do a change that I'm not 100% sure I can follow the consequences of and test.